importer/archive.go

```diff
@@ -70,3 +72,26 @@ func (a *Archive) Close() {
 		log.Warn(a.Context, "cannot close zip file", log.Data{"error": e.Error()})
 	}
 }
+
+func IsRegular(f *zip.File) bool {
+	return f.Mode().IsRegular() && f.Name[0] != '.' && f.Name != "__MACOSX"
```

Comment:
sorry dont get this.. mac specific code ? will that be ok ?

Reply:
yeah if someone on MacOS right clicks directory and hits compress - this file is created. There is a stackoverflow link in test actually - but ill move it here. there maybe some Windows special files too but i dont have a windows machine 🙈